src/main/java/com/example/securitynew/ServletInitializer.java

```diff
@@ -9,5 +9,4 @@ public class ServletInitializer extends SpringBootServletInitializer {
     protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
```

Comment:
Why do you need ServletInitializer class? In our case it is redundant.

Reply:
not fixed